hub/auto/infer.py

```diff
@@ -35,6 +35,8 @@ def _find_root(path):
     subs = [
         sub for sub in subs if os.path.isdir(sub)
     ]  # only keep directories (ignore files)
+    if len(subs) == 0:
```

Comment:
you can condense this to be:

```
if len(subs) >= 0:
    return path
```

Reply:
Sure! In the changes I've pushed, I handled the case where `len(subs)` was equal to 1, as all the other cases require `path` to be returned. All the tests pass on this change.